[CI Failure]:

### Name of failing test

models/multimodal/processing/test_common.py::test_processing_correctness[1.0-32-0.3-internlm/Intern-S1]

### Basic information

- [ ] Flaky test
- [ ] Can reproduce locally
- [ ] Caused by external libraries (e.g. bug in `transformers`)

### 🧪 Describe the failing test

Test failure introduced by a GPT-OSS PR

[2025-08-06T21:09:08Z] FAILED models/multimodal/processing/test_common.py::test_processing_correctness[1.0-32-0.3-internlm/Intern-S1] - AttributeError: 'GotOcr2ImageProcessorFast' object has no attribute 'get_number_of_image_tokens'. Did you mean: 'get_number_of_image_patches'?


### 📝 History of failing test

https://buildkite.com/organizations/vllm/analytics/suites/ci-1/tests/396bc3d2-b2a6-8b15-b051-5a341f82961c?branch=main&period=14days

appears to have been introduced by #223278


[test run history](https://buildkite.com/organizations/vllm/analytics/suites/ci-1/runs/e7f09665-d87e-8618-a7a1-fb8055c9fd40)

### CC List.

_No response_

--- a/vllm/model_executor/models/interns1.py
+++ b/vllm/model_executor/models/interns1.py
@@ -161,7 +161,7 @@ class InternS1ProcessingInfo(BaseProcessingInfo):
         if not isinstance(processor, GotOcr2ImageProcessorFast):
             raise ValueError(f'GotOcr2ImageProcessorFast is expected but got '
                              f'{type(processor)}')
-        num_image_patches = processor.get_number_of_image_tokens(
+        num_image_patches = processor.get_number_of_image_patches(
             image_height, image_width, images_kwargs=dict())
         num_image_tokens = self.get_hf_processor(
         ).image_seq_length * num_image_patches

